src/logic/calculate.js

```diff
@@ -0,0 +1,39 @@
+import operate from './operate';
+
+export default function calculate(calculate, buttonName) {
+  let { total, next, operation } = calculate;
+  const symbols = ['-', '+', '÷', 'x'];
+  const numbers = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+
+  if (buttonName === 'AC') {
+    total = null;
+    next = null;
+    operation = null;
+  }
+
+  if (buttonName === '+/-') {
+    total = (total * (-1)).toString();
+    next = (next * (-1)).toString();
+  }
```

Comment:
Both total and next should not change. You should decide on which one you need to alter the sign.